lib/iex/test/iex/autocomplete_test.exs

```diff
@@ -399,4 +399,41 @@ defmodule IEx.AutocompleteTest do
     assert expand('take(') == {:yes, '', ['take(enumerable, amount)']}
     assert expand('derive(') == {:yes, '', ['derive(protocol, module, options \\\\ [])']}
   end
+
+  @tag :tmp_dir
+  test "path completion inside strings", %{tmp_dir: dir} do
+    File.cd!(dir)
```

Comment:
We can't use `File.cd!` on async: true as that is global. Furthermore, we should always revert back. If we need File.cd! for the tests, then we should move it to a separate module that does not have async: true.

Reply:
I went with actually completely removing `File.cd!`, as we don't actually need it to test the relative path completion. Looks like the tests are not passing on CI though, will investigate what's happening later today.